Modules/_testsinglephase.c

```diff
@@ -137,13 +137,10 @@ init_module(PyObject *module, module_state *state)
 
     double d = _PyTime_AsSecondsDouble(state->initialized);
     PyObject *initialized = PyFloat_FromDouble(d);
-    if (initialized == NULL) {
-        return -1;
-    }
     if (PyModule_AddObjectRef(module, "_module_initialized", initialized) != 0) {
         return -1;
     }
-
+    Py_XDECREF(initialized);
```

Comment:
Please follow my suggestion instead. It is way easier to keep track of references using that idiom.